src/Text/Pandoc/Readers/Markdown.hs

```diff
@@ -1732,6 +1733,42 @@ source = do
 linkTitle :: PandocMonad m => MarkdownParser m Text
 linkTitle = quotedTitle '"' <|> quotedTitle '\''
 
+-- Github wiki style link, with optional title
+-- syntax documented under https://help.github.com/en/github/building-a-strong-community/editing-wiki-content
+githubWikiLink :: PandocMonad m => MarkdownParser m (F Inlines)
+githubWikiLink = try $ do
+  guardEnabled Ext_wikilinks
+  string "[["
+  inlinedContents <- option (return $ B.fromList [])
```

Comment:
Note that `B.fromList []` is just `mempty` which would be clearer and faster.

Reply:
If you want to get both the inline contents and the raw contents, you can use the `withRaw` combinator (T.P.Parsing).

Reply:
Actually, I do not know if I did correctly, but it broke the tests